tests/middlewareTypes.test.tsx

```diff
@@ -259,18 +259,53 @@ describe('counter state spec (double middleware)', () => {
     __DEV__ = savedDEV
   })
 
-  it('devtools & immer', () => {
+  it('immer & devtools', () => {
     __DEV__ = false
     const useBoundStore = create<CounterState>()(
-      devtools(
-        immer((set, get) => ({
-          count: 0,
-          inc: () =>
-            set((state) => {
-              state.count = get().count + 1
-            }),
-        })),
-        { name: 'prefix' }
+      immer(
+        devtools(
+          (set, get) => ({
+            count: 0,
+            inc: () =>
+              set((state) => {
+                state.count = get().count + 1
+              }),
+          }),
+          { name: 'prefix' }
+        )
+      )
+    )
+    const TestComponent = () => {
+      useBoundStore((s) => s.count) * 2
+      useBoundStore((s) => s.inc)()
+      useBoundStore().count * 2
+      useBoundStore().inc()
+      useBoundStore.getState().count * 2
+      useBoundStore.getState().inc()
+      useBoundStore.setState({ count: 0 }, false, 'reset')
+      return <></>
+    }
+    TestComponent
+  })
+
+  it('immer & devtools, action with type and payload', () => {
```

Comment:
My suggestion was to add action in the above `set` call itself xD, another test case wasn't necessary, but I think it doesn't hurt much to have one, so it's okay I guess.

Reply:
Haha, okay xD I will change it.